extensions-core/multi-stage-query/src/main/java/org/apache/druid/msq/exec/WorkerImpl.java

```diff
@@ -349,6 +361,8 @@ && handleResultsReady(kernelHolder, controllerClient)) {
       }
     }
 
+    context.emitMetric("query/time", stopwatch.millisElapsed());
```

Comment:
This one should also have the same metric dimensions as the controller `query/time`.